docs/README.md

```diff
@@ -1,355 +1,166 @@
-# fluidframework-docs
+# Website
 
-This is the code and content for <https://fluidframework.com>.
+This website is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
 
-The site is generated using [Hugo](https://gohugo.io/).
+## Dependency installation
 
-## Website quick start
-
-To quickly get started previewing the website's contents locally, open the docs folder in a terminal and install the dependencies using npm.
-
-```bash
-npm i -g pnpm
-cd docs
-pnpm install
+```shell
+pnpm i
```

Comment:
No need to install pnpm any more?

Reply:
If we have to add something like it, I think `corepack enable` would be the better option now? I'm not sure exactly _what_ makes it work in the client release group, but probably works here as well? (or we should make it work).

Reply:
I actually missed that I'd removed that step. For simplicity, I'll just note that we leverage pnpm, and so devs will need to install pnpm if they haven't.